### PR TITLE
Fix source typos

### PR DESCRIPTION
**Pull request by luzpaz**
Changes to API

### CODE REVIEW

File: kivy/lib/pango/pangoft2.pxi [hunk 1]
```diff
@@ -354,7 +354,7 @@ cdef extern from "pango/pango-layout.h" nogil:
     void pango_layout_set_height(PangoLayout *layout, int height)
     int pango_layout_get_height(PangoLayout *layout)
     void pango_layout_set_spacing(PangoLayout *layout, int spacing)
-    int pango_layout_get_spacing(PangoLayout *laout)
+    int pango_layout_get_spacing(PangoLayout *layout)
```
Comment (luzpaz) [inline kivy/lib/pango/pangoft2.pxi:357]: couldn't find this typo upstream. I believe it is a typo. I have not tested it.

File: kivy/tools/report.py [hunk 2]
```diff
@@ -60,8 +60,8 @@ def send_report(dict_report):
                 "content": "\n".join(dict_report['Configuration']),
                 "type": 'text'
             },
-            "Input Availablity.txt": {
-                "content": "\n".join(dict_report['InputAvailablity']),
+            "Input Availability.txt": {
+                "content": "\n".join(dict_report['InputAvailability']),
```
Comment (luzpaz) [inline kivy/tools/report.py:64]: I'm just bringing these to you attention. I can revert if necessary.

Comment (tshirtman) [reply kivy/tools/report.py:64]: i think it's fine to fix this one as well, it's just a tool, not part of the kivy api, not worth keeping an error around for the sake of backward compatibility :)

**Review by tshirtman — APPROVED** [review]
I don't think there is any issue with fixing these as well, sure, there is the off-chance that someone used the arguments as keywords in their code, but as these were documented as positional, i doubt that's the case, especially with misspelling, that would have been pretty unnatural.

**Review by matham — APPROVED** [review]

**Pull request merged**